doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-marshalling-ref.adoc

```diff
@@ -0,0 +1,4 @@
+[id='kie-server-commands-marshalling-ref_{context}']
+= {KIE_SERVER} and KIE container commands marshalling
+
+*<@Maciej/@Tiho: We need here whatever the equivalent {KIE_SERVER} command marshalling requirements are, like <<runtime-commands-marshalling-ref_kie-apis>>.>*
```

Comment:
as commented above this marshalling of commands should not be added. Its an internal implementation and customers should rely on kie server client or kie server api to do the marshalling instead of another set of classes like described in the runtime commands section

Reply:
Done.